projects/lua/README.md

```diff
@@ -0,0 +1,40 @@
+# Lua
+
+[Lua][lua-homepage] is a powerful, efficient, lightweight, embeddable
+scripting language. Lua is dynamically typed, runs by interpreting bytecode
+with a register-based virtual machine, and has automatic memory management with
+incremental garbage collection, making it ideal for configuration, scripting,
+and rapid prototyping. Lua is designed, implemented, and maintained by a team
+at PUC-Rio, the Pontifical Catholic University of Rio de Janeiro in Brazil.
+
+## Build Docker
+
+    $ sudo docker build -t oss-sydr-fuzz-lua .
+
+## Run Hybrid Fuzzing
+
+Unzip Sydr (`sydr.zip`) in `projects/lua` directory:
+
+    $ unzip sydr.zip
+
+Run docker:
+
+    $ sudo docker run --privileged --network host -v /etc/localtime:/etc/localtime:ro --rm -it -v $PWD:/fuzz oss-sydr-fuzz-lua /bin/bash
+
+Change directory to `/fuzz`:
+
+    # cd /fuzz
+
+Run hybrid fuzzing:
+
+    # sydr-fuzz -c luaL_loadstring.toml run
+
+Collect and report coverage:
+
+    # sydr-fuzz -c luaL_loadstring.toml cov-report
```

Comment:
Maybe add description of generating html report?

Reply:
I don't know what for duplicate sydr manual in a README, but I'm added:

```diff
--- a/projects/lua/README.md
+++ b/projects/lua/README.md
@@ -33,6 +33,10 @@ Collect and report coverage:
 
     # sydr-fuzz -c luaL_loadstring.toml cov-report
 
+Building HTML report:
+
+       $ sydr-fuzz -c luaL_loadstring.toml cov-show -- -format=html > index.html
+
 ## Hybrid Fuzzing with AFL++
 
     # sydr-fuzz -c luaL_loadstring-afl++.toml run
```